airflow/config_templates/default_test.cfg

```diff
@@ -48,6 +48,7 @@ dag_concurrency = 16
 dags_are_paused_at_creation = False
 fernet_key = {FERNET_KEY}
 non_pooled_task_slot_count = 128
+non_pooled_backfill_task_slot_count = 128
```

Comment:
```suggestion
```

Not needed due to how config loading works.

Reply:
fixed.